client/state/login/magic-login/actions.js

```diff
@@ -70,9 +81,58 @@ export const fetchMagicLoginRequestEmail = email => dispatch => {
 	} );
 };
 
+export const authError = error => {
+	return {
+		type: MAGIC_LOGIN_REQUEST_AUTH_ERROR,
+		error,
+	};
+};
+
+export const authSuccess = () => {
+	return {
+		type: MAGIC_LOGIN_REQUEST_AUTH_SUCCESS,
+	};
+};
+
+export const fetchMagicLoginAuthenticate = ( email, token, tt ) => dispatch => {
+	dispatch( { type: MAGIC_LOGIN_REQUEST_AUTH_FETCH } );
+
+	const postData = {
+		client_id: config( 'wpcom_signup_id' ),
+		client_secret: config( 'wpcom_signup_key' ),
+		email,
+		token,
+		tt,
+	};
+
+	request
```

Comment:
Perhaps. It should be as similar as possible to the "regular" auth network functionality which is yet to be implemented afaik. It's probably also worthwhile to see if we can use [wpcom-xhr-request](https://github.com/Automattic/wpcom-xhr-request/) to abstract instead of calling superagent / request directly.

I'll capture this as something to investigate in #12677